### PR TITLE
serializers: fix templating bugs

### PR DESCRIPTION
**Pull request by vbalbp**
Signed-off-by: vbalbp <vbalbp@gmail.com>

### CODE REVIEW

File: inspirehep/modules/records/serializers/schemas/latex/__init__.py [hunk 1]
```diff
@@ -41,6 +41,14 @@ class LatexSchema(Schema):
     texkeys = fields.Method('get_texkey')
     today = fields.Method('get_current_date')
 
+    def get_citations(self, data):
+        citations = data.get('citation_count')
```
[the remark this answers is not in the document]
Comment (drjova) [reply inspirehep/modules/records/serializers/schemas/latex/__init__.py]: ``data`` is always an InspireRecord? For search it comes from ES AFAIK

[the remark this answers is not in the document]
Comment (michamos) [reply inspirehep/modules/records/serializers/schemas/latex/__init__.py]: @drjova that's the whole point. If it's an `InspireRecord` it has a `get_citations_count` method but no `citation_count` key, if it comes from ES it's the other way around. This forces every serializer to deal with both cases explicitly.

**Comment by vbalbp** [comment]
ci test please

**Comment by michamos** [comment]
Jenkins is still failing

**Pull request merged**